x/bep3/keeper/grpc_query.go

```diff
@@ -0,0 +1,153 @@
+package keeper
```

Comment:
I'd add tests covering the query methods. Particularly around the filtered paginate logic.